README.md

```diff
@@ -1,2 +1,197 @@
-# CoinStats-WebApp-Metrics-M3-Capstone
+<a name="readme-top"></a>
+<div align="center">
+  <img src="./luis-logo.png" alt="logo" width="30%"  height="auto" />
+</div>
+
+<br/>
+
+
+<h1>CryptoCoins Stats</h1>
+
+<a name="readme-top"></a>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 coinmarket](#-coinmarket)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [👥 Author ](#-author-)
+  - [🔭 Future Features ](#-future-features-)
+  - [Walkthrough ](#walkthrough-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 coinmarket<a name="about-project"></a>
+
 The CoinStats is a SPA fully responsive, that shows information from the top 100 cryptocoins. The app displays the top 100 and the user can select a cryptocoin to see more information
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+- HTML
+- JS
+- CSS
+- React
+- Redux Toolkit (RTK)
+
+<details>
+  <summary>Client</summary>
+  - HTML5<br>
+  - JS<br>
+  - CSS3<br>
+  - React.js<br>
+  - Redux.js<br>
+  - Jest.js
+</details>
+
+<details>
+  <summary>Server</summary>
+    - Null
+</details>
+
+<details>
+<summary>Database</summary>
+    - Null
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **Coinmarket using React and Redux**
+- **Client-side routing using React Router V6**
+- **Gitflow is used correctly**
+- **Following best practices for HTML, CSS, JS**
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+🚀 Live demo of this project [here](https://cryptocoins-stats.onrender.com/)
+
+🚀 Video Info of this project [here](https://www.loom.com/share/25395352af5441a890adfc54ffc13f7c)
```

Comment:
- [ ] @lerfast Great work adding the video presentation, however, you forgot to highlight a piece of the code the student finds interesting or is proud of making.Kindly consider ensuring your presentation fulfills all the project requirements shown below
![image](https://github.com/lerfast/coinstats-webapp-metrics-m3-capstone/assets/61408860/f3dc6e5f-25be-46d8-aef6-64c391951a3a)

Reply:
![image](https://github.com/lerfast/coinstats-webapp-metrics-m3-capstone/assets/111823606/37454cdd-0f32-4690-92fc-736866875dd9)
I recorded again the video, and I am sorry I fell nervous in front of a camera, I have to work with that, and I hope that this time it makes it clearer.